client/state/sites/constants.js

```diff
@@ -5,7 +5,6 @@ export const SITE_REQUEST_FIELDS = [
 	'icon',
 	'is_multisite',
 	'is_private',
-	'is_coming_soon',
```

Comment:
It's not clear to me:

1. why this is in master already
1. why we're removing it here

Reply:
It was merged to master in this commit: https://github.com/Automattic/wp-calypso/pull/38803/files

And we should not be removing it here - great spot!